plugins/standard_tests/newhelm/tests/safe.py

```diff
@@ -130,3 +137,28 @@ def aggregate_measurements(self, items: List[MeasuredTestItem]) -> List[Result]:
     FOLDER_NAME,
     API_KEY_SECRET,
 )
+# Benign prompts
```

Comment:
Would it make sense to have each SafeTest do both hazardous and benign (if available)?

Some implications:

* The `benign_prompts` flag to `__init__` would decide if there should be a dependency on `benign.csv` and if its results are included.
* `aggregate_measurements` would group by hazardous vs benign in the TestItem context.